test/drafts/ERC20Migrator.test.js

```diff
@@ -0,0 +1,142 @@
+const { assertRevert } = require('../helpers/assertRevert');
+
+const ERC20Mock = artifacts.require('ERC20Mock');
+const ERC20Mintable = artifacts.require('ERC20Mintable');
+const ERC20Migrator = artifacts.require('ERC20Migrator');
+
+const BigNumber = web3.eth.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('ERC20Migrator', function ([_, owner, recipient, anotherAccount]) {
+  const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+  const totalSupply = 200;
+
+  beforeEach('deploying tokens', async function () {
+    this.legacyToken = await ERC20Mock.new(owner, totalSupply);
+    this.migrator = await ERC20Migrator.new(this.legacyToken.address);
+    this.newToken = await ERC20Mintable.new();
+  });
+
+  describe('beginMigration', function () {
+    it('rejects zero address for token', async function () {
```

Comment:
As per #1157, this should read something like 'reverts with a null new token address'